src/org/openstreetmap/josm/data/osm/Relation.java

```diff
@@ -426,6 +426,11 @@ public List<OsmPrimitive> getMemberPrimitivesList() {
         return Utils.transform(getMembers(), RelationMember::getMember);
     }
 
+    @Override
+    public List<OsmPrimitive> getChildren() {
+        return getMemberPrimitivesList();
+    }
+
```

Comment:
See comment in `IPrimitive`. It should either be there or `IRelation`. If you put it in `IRelation`. In `IRelation`, it might be a bit tricky to have it return the appropriate primitive class without breaking backwards compatibility, so it _might_ be OK there.